apps/aecore/src/aec_db.erl

```diff
@@ -86,6 +88,7 @@
 -record(aec_blocks             , {key, txs}).
 -record(aec_headers            , {key, value, height}).
 -record(aec_contract_state     , {key, value}).
+-record(aec_call_state         , {key, value}).
```

Comment:
I expect starting with old DB will result in error `missing_table` so I suggest no versioning changes in aec_db needed. Ok.

Reply:
I guess we will start from new genesis and reset most things in this release.